code/modules/clothing/head/helmet.dm

```diff
@@ -80,6 +77,12 @@
 	can_toggle = 1
 	toggle_cooldown = 20
 	activation_sound = 'sound/items/WEEOO1.ogg'
+	
+/obj/item/clothing/head/helmet/justice/attack_self()

```

Comment:
But the system was placed on /helmet (it may even be lower) to ALLOW future sounds.
Rename the var if you must, but at present this is a constant sound system.